@xen-orchestra/lite/src/components/ui/UiCardTitle.vue

```diff
@@ -40,8 +43,16 @@ defineProps<{
   --section-title-right-weight: 700;
 
   &.subtitle {
-    border-bottom: 1px solid var(--color-extra-blue-base);
+    margin-bottom: 1rem;
+    --section-title-left-size: 1.5rem;
+    --section-title-left-color: var(--color-blue-scale-300);
+    --section-title-left-weight: 400;
+  }
 
+  &.subtitle-with-underline {
+    margin-top: 2rem;
+    margin-bottom: 1rem;
+    border-bottom: 1px solid var(--color-extra-blue-base);
```

Comment:
The name is confusing.
With this name i expect having exactly the same styles than for `subtitle` class with an extra `border-bottom`.

Reply:
I think `UiCardTitle` needs a major overhaul because it becomes too hard to understand.

The idea behind levels `1`, `2` and `3` was to have 3 different styles, nothing more.

If now we need to deal with an extra (optional) `subtitle-class` prop, it adds a layer of complexity.

Even worse, `subtitle` and `subtitle-with-underline` not only adds an underline but also change the underlying style of the component sections.

Additionally, it means that it's possible to pass level="1" with class="subtitle" which wouldn't make sense.

I'll take some time to figure this out and find a proper solution.

Reply:
I tested the component and see that in fact, `level` prop doesn't change the style, only the tag.

I think the `subtitle-class` prop should be removed and the style should be based on the `level` prop.

`:level="1"` => Default title
`:level="2"` => Subtitle
`:level="3"` => Subtitle with underline